app/models/target.rb

```diff
@@ -34,8 +34,6 @@ def get_progress
         conference.registrations.where('created_at < ?', due_date).count
       when Target.units[:program_minutes]
         conference.current_program_minutes
-      else
-        0
```

Comment:
> Not sure you should remove that!

A simple answer would be that I couldn't write test for that case, hence I removed it. If we can't write test for something, we shouldn't have it in our code.

More detailed one:
We have `switch` statement over `unit`. `unit` can never have values other than first three cases as we validate `unit` for inclusion in:
```Ruby

  def self.units
    {
      registrations: 'Registration',
      submissions: 'Submission',
      program_minutes: 'Program minute'
    }
  end

 validates :unit, allow_nil: false, inclusion: { in: Target.units.values }
```